freight/api/task_index.py

```diff
@@ -137,10 +118,10 @@ def post(self):
             if not sha:
                 return self.error('Invalid ref', name='invalid_ref', status_code=400)
         else:
-            try:
-                sha = vcs_backend.get_sha(ref)
-            except vcs.UnknownRevision:
```

Comment:
Yes it was, I will investigate pre-c64ae92fb8f827626eece01bd21843beba12216e so and revert the change.

```
[git:master] thoas belgariade ~/Sites/Python/freight
$ git show-ref --hash=0 c64ae92fb8f827626eece01bd21843beba12216e
```

shows nothing instead of

```
[git:master] thoas belgariade ~/Sites/Python/freight
$ git describe --always --abbrev=0 c64ae92fb8f827626eece01bd21843beba12216e
c64ae92fb8f827626eece01bd21843beba12216e
[git:master] thoas belgariade ~/Sites/Python/freight
$ git describe --always --abbrev=0 master
0ccb49b90a58861e421babf8d0bf480d3314838d
```